libs/partners/unstructured/docs/document_loaders.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
This file was auto-generated when initializing the partner package. @efriis, would it be okay to remove this file and just depend on the existing documentation files (`docs/docs/integrations/providers/unstructured.mdx` and `docs/docs/integrations/document_loaders/unstructured_file.ipynb`)? Those have been updated to reference the SDK Loaders instead of the API Loaders.

Reply:
yes! And feel free to remove the whole unstructured/docs directory. You have to move these files into the respective directories under root/docs!